pkg/cluster/kubeclient/status.go

```diff
@@ -0,0 +1,22 @@
+package kubeclient
+
+import (
+	"context"
+	"encoding/json"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (u *kubeclient) GetPodsStatus(ctx context.Context, namespaces []string) ([]byte, error) {
```

Comment:
suggest you don't take in a namespaces argument.  List the namespaces, and output pods for the following namespaces:
* default
* openshift
* anything with prefix "openshift-"
* anything with prefix "kube-"

please put the function which chooses if a namespace is "of interest" in a shared module and refactor everywhere else in the codebase where you find the text `"kube-"`

Reply:
MSFT is checking only `kube-system` and `openshift-node` namespaces. Please confirm if you want me to follow your suggestion nonetheless.

Reply:
yes, I do want you to follow my suggestion :)

Reply:
done